components/Tabs/Tabs.js

```diff
@@ -1,51 +1,63 @@
-
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
-    
+    this.element = element;
+
     // Get the custom data attribute on the Link
-    // this.data;
-    
+    this.data = this.element.dataset.tab;
+
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
+    this.itemElement = document.querySelector(
+      `.tab-item[data-tab="${this.data}"]`
+    );
+
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
-    
+    this.tabItem = new TabItem(this.itemElement);
+    //Why itemElement???
     // Add a click event listener on this instance, calling the select method on click
-
+    this.element.addEventListener('click', this.select(this));
```

Comment:
missing some syntax here.... 

two ways to bind....

1. arrow function

2. with bind method

Look up syntax for both and correct.